evoting/lib/container.go

```diff
@@ -43,15 +43,19 @@ type Mix struct {
 	Ballots []*Ballot // Ballots are permuted and re-encrypted.
 	Proof   []byte    // Proof of the shuffle.
 
-	Node string // Node signifies the creator of the mix.
+	Node      string      // Node signifies the creator of the mix.
+	PublicKey kyber.Point // Public key of the node generating the mix
```

Comment:
I prefer having this as the `ServerIdentity.ID`, so you can look up the public key in the roster of the nodes. Else you might get a rogue signature with a random private/public key. Also for the `Partial` structure.